package.json

```diff
@@ -99,5 +99,8 @@
     "webpack-dev-server": "^2.9.4",
     "webpack-merge": "^4.1.0",
     "zone.js": "0.8.18"
+  },
+  "dependencies": {
```

Comment:
Do 'compat' really needed?

Reply:
For this first merge to go smoothly, I’d leave it. We can see to suppressing it if all works without it.